src/main/java/mServer/crawler/sender/arte/ArteVideoDetailsDeserializer.java

```diff
@@ -0,0 +1,194 @@
+package mServer.crawler.sender.arte;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.util.Calendar;
+import mServer.tool.DateWithoutTimeComparer;
+import org.apache.commons.lang3.time.FastDateFormat;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ArteVideoDetailsDeserializer implements JsonDeserializer<ArteVideoDetailsDTO> {
+
+    /*
+     * Grobe Struktur des Json's:
+     * {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)